content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -107,6 +108,37 @@ ValidatingAdmissionPolicy 'demo-policy.example.com' with binding 'demo-binding-t
 
 The above provides a simple example of using ValidatingAdmissionPolicy without a parameter configured.
 
+#### Validation actions
+
+Each `ValidatingAdmissionPolicyBinding` must specify one or more
+`validationActions` to declare how `validations` of a policy are enforced.
+
+The supported `validationActions` are:
+
+- `Deny`: Validation failure results in a denied request.
+- `Warn`: Validation failure is reported to the request client
+  as a [warning](/blog/2020/09/03/warnings/).
+- `Audit`: Validation failure is included in the audit event for the API request.
+
+For example, to both warn clients about a validation failure and to audit the
+validation failures, use:
+
+```yaml
+validationActions: [Warn, Audit]
+```
+
+`Deny` and `Warn` may not be used together since this combination
+needlessly duplicates the validation failure both in the
+API response body and the HTTP warning headers.
+
+A `validation` that evaluates to false is always enforced according to these
+actions. Failures defined by the `failurePolicy` are enforced
```

Comment:
Do we wanna mention failurePolicy in ValidatingAdmissionPolicy since a binding.failurePolicy is potentially could be added?

Reply:
For binding failurePolicy, let's wait till we add it to document it.

For this PR, the interaction between validationActions and failurePolicy is what needed to be added. Let me know if you see any gaps in how it was documented.